embassy-stm32/src/rcc/bd.rs

```diff
@@ -1,12 +1,19 @@
+#[allow(dead_code)]
+#[derive(Clone, Copy)]
+pub enum LseCfg {
+    Oscillator(#[cfg(any(rtc_v2f7, rtc_v2h7, rtc_v2l0, rtc_v2l4))] LseDrive),
```

Comment:
Let's not cfg this, rather, make `Oscillator(MediumHigh)` be the default and do nothing for chips that don't support it.